app/Nova/Travel.php

```diff
@@ -78,315 +80,387 @@ class Travel extends Resource
+                    ->required()
+                    ->rules('sometimes')
+                    ->min(0)
+                    ->max(100)
+                    ->hideFromIndex(),
+
+                Heading::make('Accounting Information')
+                    ->dependsOn(
+                        ['forms'],
+                        static function (Heading $field, NovaRequest $request, FormData $formData): void {
+                            if (
+                                self::showFieldOnForms(
+                                    $formData,
+                                    \App\Models\Travel::TRAVEL_INFORMATION_FORM_KEY,
+                                    \App\Models\Travel::AIRFARE_REQUEST_FORM_KEY
+                                )
+                            ) {
+                                $field->show();
+                            }
+                        }
+                    )
+                    ->hide()
+                    ->showOnDetail(fn (): bool => $this->showFieldOnDetail(
+                        \App\Models\Travel::TRAVEL_INFORMATION_FORM_KEY,
+                        \App\Models\Travel::AIRFARE_REQUEST_FORM_KEY
+                    )),
+
+                Text::make('Workday Account Number', 'tar_project_number')
+                    ->dependsOn(
+                        ['forms'],
+                        static function (Text $field, NovaRequest $request, FormData $formData): void {
+                            if (
+                                self::showFieldOnForms(
+                                    $formData,
+                                    \App\Models\Travel::TRAVEL_INFORMATION_FORM_KEY,
+                                    \App\Models\Travel::AIRFARE_REQUEST_FORM_KEY
+                                )
+                            ) {
+                                $field->show()
+                                    ->rules(
+                                        'required',
+                                        'max:255',
+                                        'in:CE0339,DE00007513,GTF250000211' // agency, SGA, ME GTF
+                                    );
+                            }
+                        }
+                    )
+                    ->showOnDetail(fn (): bool => $this->showFieldOnDetail(
+                        \App\Models\Travel::TRAVEL_INFORMATION_FORM_KEY,
+                        \App\Models\Travel::AIRFARE_REQUEST_FORM_KEY
+                    ))
+                    ->required()
+                    ->hide()
+                    ->rules('sometimes')
+                    ->help(
+                        'Ask the treasurer for the correct value for this field.'
+                    )
+                    ->hideFromIndex(),
+
+                Select::make('Department', 'department_number')
+                    ->options(DepartmentNumbers::DESCRIPTIONS)
+                    ->displayUsingLabels()
+                    ->searchable()
```

Comment:
This is I think just using basic substring search on the frontend. We can make this fuzzy search with a model like we did for airports but that feels like overkill.